src/Diagnostics.DataProviders/DataProviders/GeoMasterDataProvider.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using System.Threading;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using Diagnostics.ModelsAndUtils.Models;
+
+namespace Diagnostics.DataProviders
+{
+
+    public class GeoMasterDataProvider : DiagnosticDataProvider, IDiagnosticDataProvider
+    {
+        private readonly IGeoMasterClient _geoMasterClient;
+        private GeoMasterDataProviderConfiguration _configuration;
+        
+        public GeoMasterDataProvider(OperationDataCache cache, GeoMasterDataProviderConfiguration configuration) : base(cache)
+        {
+            _configuration = configuration;
+            _geoMasterClient = InitClient();
+        }
+
+        private IGeoMasterClient InitClient()
+        {
+            IGeoMasterClient geoMasterClient;
```

Comment:
Create GeoMasterMockClient for testing

Reply:
I have never written test projects so this is something I would like to understand and post that I can add the right clients. This is something I was planning to discuss next week anyways...